### ISSUE TITLE
OCR and player problem - Mono Ubuntu 18.04

### ISSUE THREAD

**Issue by Gitraffe** [SubtitleEdit/subtitleedit#3155]
1.
OCR seem's to works with English dictionary with some bugs. (Example if there is unknown word program want's edit whole text, not just unknown word.)
I have tried to use Finnish dictionary, but I can't make it to work.

> Terminal:
> mono ~/Desktop/SubtitleEdit/SubtitleEdit.exe
> Gtk-Message: 06:35:13.929: Failed to load module "canberra-gtk-module"
> error: /home/xx/Desktop/SubtitleEdit/Dictionaries/fi_FI.aff: cannot open

I think Finnish is not using those .aff files? Is there any way to make it work with mono?

2. 
How I can install videoplayer/engine when using mono? Everything is greyed.
Would like to use MPV if possible.

**Comment by niksedk** [comment]
Sorry, I don't have Ubuntu installed atm...

I don't think Finnish is supported by aff/dic Hunspell file pair.
SE uses https://github.com/voikko on windows - so I guess getting Finnish spellcheck working on Linux will require some coding.

"libmpv" would probably be the best solution for video player.

Also check this: https://github.com/SubtitleEdit/subtitleedit/issues/3142

**Comment by Gitraffe** [comment]
Sounds like it would be better to use Wine if Finnish is required... With wine it worked untill yesterday.
I don't know what happened, but OCR language option is blank (sup to srt) I can download new languages to folder, but list stays blank.
I was so frustrated yesterday, so I installed fresh Ubuntu to my system, but it wouldn't solve this problem.

PS: I emailed to you about week ago of my video playback problem, have you checked your email lately?
